app/components/oral_history/downloads_list_component.html.erb

```diff
@@ -17,14 +17,32 @@
   </div>
 <% end %>
 
-<% if combined_mp3_audio_url.present? && combined_derivatives_up_to_date?%>
+<% if combined_m4a_audio_url.present? && combined_derivatives_up_to_date?%>
   <h2 class="attribute-sub-head">
     <% if audio_members.count == 1 %>
       Audio File <small class="text-muted">Web-quality download</small>
     <% else %>
       Complete Interview Audio File <small class="text-muted">Web-quality download</small>
     <% end %>
   </h2>
+  <div class="combined-audio-download-links">
+    <div class="combined-audio-download-icon-container">
+      <a href="<%= combined_m4a_audio_download_url %>"
+        data-analytics-category="Work"
+        data-analytics-action="download_combined_audio_derivatives"
+        data-analytics-label="<%= work.friendlier_id %>"
+        >
+        <i class="fa fa-download" aria-hidden="true"></i>
+        Web-quality m4a
+      </a>
+    </div>
+    <div class="combined-audio-download-link-size">
+      <small><%= combined_display_m4a_audio_size %></small>
+    </div>
+  </div>
+<% end %>
+
+<% if combined_mp3_audio_url.present? #REMOVE_AFTER_MP3_TO_M4A_MIGRATION %>
```

Comment:
Should this have `&& combined_derivatives_up_to_date?` like it used to, and like the above one for `combined_m4a_audio_url` now does?

Reply:
That check is really looking up the current code in CombinedAudioDerivativeCreator.new(work).fingerprint, which should return `true` throughout the process: that method is really not looking at the derivative but at the underlying audio assets, which are not changing at any point. That said, there's no reason *not* to include the check, now that I think about it!

Reply:
Why do you say "it should return true throughout the process"? What process? The creation of the list of download links?

As I understand it, the inclusion of the mp3 URL was guarded by `if combined_derivatives_up_to_date?`, so that the mp3 URL wouldn't be included in the list of download lists offered to the user if it was not up to date. 

I believe without this check, if the mp3 URL (or m4a URL later) is not up to date, then it will still be included in the list of download links to offered to the user. 

So it's not a no-op, it's doing something. At least that was my intention, and why it is present in the code before this PR.  I acknolwedge it's not ideal that removing it didn't cause any tests to fail!

Reply:
Yeah - no problem including the check. Just noting that "up to date" doesn't refer to the mp3 vs. m4a distinction: as long as the published audio assets aren't changed, any and all combined audio derivatives are considered "up to date".
The check method would only hide the mp3 if someone were to change the audio assets in some way before we removed the old derivatives.

Reply:
The fingerprint code is functioning as intended, did not intend to suggest that. And everything else you said is correct: 
"if the original files change, and old combined derivative(s) are present, they can be recognized as out of date, and not shown.". We're on the same page.